lib/osf-components/addon/components/search-page/template.hbs

```diff
@@ -241,25 +241,38 @@ as |layout|>
             {{/each}}
         {{/if}}
         <div local-class='pagination-buttons'>
-            {{#if this.firstPageCursor}}
+            {{#if this.prevPageCursor}}
                 <Button
-                    {{on 'click' (fn this.switchPage this.firstPageCursor)}}
+                    {{on 'click' (fn this.switchPage this.prevPageCursor)}}
                 >
-                    {{t 'search.first'}}
+                    <p>
+                        {{t 'search.previous-pagination-button'}}
+                    </p>
                 </Button>
             {{/if}}
-            {{#if this.prevPageCursor}}
+            {{#if this.firstPageCursor}}
                 <Button
-                    {{on 'click' (fn this.switchPage this.prevPageCursor)}}
+                    {{on 'click' (fn this.switchPage this.firstPageCursor)}}
                 >
-                    {{t 'search.prev'}}
+                    <p>
+                        {{t 'search.first-pagination-button'}}
+                    </p>
                 </Button>
             {{/if}}
+            {{#if this.nextPageCursor}}
+                <div>
+                    <p>
+                        {{t 'search.more-pagination-button'}}
+                    </p>
+                </div>
+            {{/if}}
             {{#if this.nextPageCursor}}
                 <Button
                     {{on 'click' (fn this.switchPage this.nextPageCursor)}}
                 >
-                    {{t 'search.next'}}
+                    <p>
+                        {{t 'search.next-pagination-button'}}
+                    </p>
```

Comment:
During one of the scrum discussion, we have decided to have three pagination buttons (First, Prev, Last). So it is okay to not follow the mockup here.

Reply:
Ok, that sounds good.